src/sentry/static/sentry/app/actionCreators/globalSelection.tsx

```diff
@@ -113,22 +113,35 @@ export function initializeUrlState({
   const orgSlug = organization.slug;
   const query = pick(queryParams, [URL_PARAM.PROJECT, URL_PARAM.ENVIRONMENT]);
   const hasProjectOrEnvironmentInUrl = Object.keys(query).length > 0;
-  const parsed = getStateFromQuery(queryParams, {allowAbsoluteDatetime: showAbsolute});
+  const parsed = getStateFromQuery(queryParams, {
+    allowAbsoluteDatetime: showAbsolute,
+    allowEmptyPeriod: true,
+  });
+  const {datetime: defaultDateTime, ...retrievedDefaultSelection} = getDefaultSelection();
+  const {datetime: customizedDefaultDateTime, ...customizedDefaultSelection} =
+    defaultSelection || {};
 
   let globalSelection: Omit<GlobalSelection, 'datetime'> & {
     datetime: {
       [K in keyof GlobalSelection['datetime']]: GlobalSelection['datetime'][K] | null;
     };
   } = {
-    ...getDefaultSelection(),
```

Comment:
I wonder if this was intended to have its datetime overrided - let's see what the test says

Reply:
Hmm I don't think it was since the utc type mismatched. But I did want to grab the default period from it